core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -51,13 +55,38 @@ angular.module('oppia').controller(
         return `${contentType} section of "${stateName}" card`;
       };
 
+      var _getThreadHandlerUrl = function(suggestionId) {
+        return UrlInterpolationService.interpolateUrl(
+          '/threadhandler/<suggestionId>', { suggestionId });
+      };
+
+      var _getThreadMessagesAsync = function(threadId) {
```

Comment:
Currently, we don't show the review messages to the contributors which makes it impossible to understand the reason for suggestion rejection. This PR adds functionality to show review messages to contributors.

Reply:
Can you document this change in the PR description? Related issue: https://github.com/oppia/oppia/issues/9464

Reply:
I had already documented all the changes in the description, I don't know where are those changes now :p, I'll re-add with the link to the corresponding issue.